cron-backup/nginx_backup.sh

```diff
@@ -4,55 +4,55 @@
 DATE1=$(date +%Y%m%d%H%M)
 DATE=$(date +%d-%m-%y_%H-%M)
 
-mkdir -p /var/lib/backup
+mkdir -p /var/lib/backup/nginx
 
 nginx_src='/nginx'
 
 if [ ! -d $nginx_src ]; then
 
-        echo "DATE:" $DATE > /nginx.txt
-        echo " " >> /nginx.txt
-        echo "DESCRIPTION: ${SOURCE_NAME}_Nginx backup" >> /nginx.txt
-        echo " " >> /nginx.txt
-        echo "STATUS: Nginx backup is Failed." >> /nginx.txt
-        echo " " >> /nginx.txt
-	echo "The source backup directory: nginx_src is not available" >> /nginx.txt
-        < /nginx.txt mail -s "${SOURCE_NAME}: Nginx Data Backup" "${CRON_BACKUP_MAIL}"
+        echo "DATE:" $DATE > /tmp/nginx.txt
+        echo " " >> /tmp/nginx.txt
+        echo "DESCRIPTION: ${SOURCE_NAME}_Nginx backup" >> /tmp/nginx.txt
+        echo " " >> /tmp/nginx.txt
+        echo "STATUS: Nginx backup is Failed." >> /tmp/nginx.txt
+        echo " " >> /tmp/nginx.txt
+	echo "The source backup directory: nginx_src is not available" >> /tmp/nginx.txt
+        < /tmp/nginx.txt mail -s "${SOURCE_NAME}: Nginx Data Backup" "${CRON_BACKUP_MAIL}"
 	exit
 else
-	tar cvzf /var/lib/backup/"${SOURCE_NAME}"_nginx_data_backup_"${DATE1}".tgz ${nginx_src}/
+	tar cvzf /var/lib/backup/nginx/"${SOURCE_NAME}"_nginx_data_backup_"${DATE1}".tgz ${nginx_src}/
 fi
 
-s3cmd sync --no-mime-magic /var/lib/backup/*nginx*.tgz s3://"${S3_BUCKET_NGINX}"/
+s3cmd put -r --no-mime-magic /var/lib/backup/nginx/ s3://"${S3_BUCKET_NGINX}"/
 
 # Moving the backup to S3 bucket
 if [ $? -eq 0 ]; then
 
-        echo "DATE:" $DATE > /nginx.txt
-        echo " " >> /nginx.txt
-        echo "DESCRIPTION: ${SOURCE_NAME}_Nginx backup" >> /nginx.txt
-        echo " " >> /nginx.txt
-        echo "STATUS: Nginx backup is Successful." >> /nginx.txt
-        echo " " >> /nginx.txt
-        echo "******* Nginx Data Backup ****************" >> /nginx.txt
-        echo " " >> /nginx.txt
-        s3cmd ls --no-mime-magic s3://"${S3_BUCKET_NGINX}"/  --human-readable | grep -i "${SOURCE_NAME}"_nginx_data | cut -d' ' -f3- | tac | head -10 | sed "s,s3:\/\/${S3_BUCKET_NGINX}\/,,g" &>> /nginx.txt
-        echo " " >> /nginx.txt
-        echo "************** END **************************" >> /nginx.txt
+        echo "DATE:" $DATE > /tmp/nginx.txt
+        echo " " >> /tmp/nginx.txt
+        echo "DESCRIPTION: ${SOURCE_NAME}_Nginx backup" >> /tmp/nginx.txt
+        echo " " >> /tmp/nginx.txt
+        echo "STATUS: Nginx backup is Successful." >> /tmp/nginx.txt
```

Comment:
"is Successful" --> "succeeded", please.

Reply:
It's done.